webapp/first_snap/content/ros2/build.yaml

```diff
@@ -0,0 +1,54 @@
+linux:
+  auto:
+    - action: Return to the root directory of the project containing your snapcraft.yaml and run snapcraft
+      command: snapcraft
+    - action: You will find a .snap file in the same directory that you ran the snapcraft command. You can inspect the contents to ensure it contains all of the application's assets
+      command: unsquashfs -l *.snap
+    - action: Install the snap
+      command: sudo snap install --devmode --dangerous *.snap
+    - action: List your installed snaps to confirm
+      command: snap list
+    - action: Run the snap
+      command: test-ros2-talker-listener-{name} -h
+    - warning: |
+        Make sure the name matches what has been used previously (i.e <code>test-ros2-talker-listener-{name}</code>)
```

Comment:
Is it intended for this to be a new li, or should it be part of the previous one? Also, kinda feels like the i.e. should be e.g.